packages/commonwealth/client/scripts/controllers/chain/cosmos/governance-v1.ts

```diff
@@ -0,0 +1,333 @@
+import type { MsgSubmitProposalEncodeObject } from '@cosmjs/stargate';
+import BN from 'bn.js';
+import type {
+  CosmosProposalState,
+  ICosmosProposal,
+  ICosmosProposalTally,
+} from 'controllers/chain/cosmos/types';
+import { CosmosToken } from 'controllers/chain/cosmos/types';
+import { CommunityPoolSpendProposal } from 'cosmjs-types/cosmos/distribution/v1beta1/distribution';
+import {
+  ProposalSDKType,
+  TallyResultSDKType,
+} from 'common-common/src/cosmos-ts/src/codegen/cosmos/gov/v1/gov';
+import { Any } from 'common-common/src/cosmos-ts/src/codegen/google/protobuf/any';
+import { ProposalStatus } from 'common-common/src/cosmos-ts/src/codegen/cosmos/gov/v1/gov';
+import { TextProposal } from 'cosmjs-types/cosmos/gov/v1beta1/gov';
+
+import type { ITXModalData } from 'models';
+import { ProposalModule } from 'models';
+import moment from 'moment';
+import type CosmosAccount from './account';
+import type CosmosAccounts from './accounts';
+import type CosmosChain from './chain';
+import type { CosmosApiType } from './chain';
+import { CosmosProposal } from './proposal';
+import { numberToLong } from 'common-common/src/cosmos-ts/src/codegen/helpers';
+
```

Comment:
See note below. This file is mainly a copy of `governance.ts` with v1 mods. I can move towards splitting paths in the original file, but this felt cleaner as a temporary solution. WDYT?

Reply:
I am happy with this as a temp solution. We want to dramatically improve these controller modules anyway, so no need to make it perfect immediately.